### ISSUE TITLE
python2 fully deprecated Jan 2020

### ISSUE THREAD

**Issue by keiranmraine** [bcgsc/xmatchview#6]
Have you considered trying to convert via [2to3](https://docs.python.org/3/library/2to3.html)

**Comment by keiranmraine** [comment]
As you dependency section states:

```
The code was developed and tested on python2.3 or 2.4 but may work with newer versions of python (not tested)
```

**Comment by lcoombe** [comment]
Hi @keiranmraine,

As you may have discovered, the latest release of xmatchview is in fact already refactored to be python3 compatible. I corrected the documentation error in #7 .

Thanks!
Lauren
